GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day2/InheritanceII.java

```diff
@@ -0,0 +1,29 @@
+package hackkerank.day2;
+
+    class Arithmetic{
+
+        int add(int x, int y){
```

Comment:
Remember, variable names and parameter names (arguments) must be meaningful:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names